app/javascript/readingList/components/TagList.jsx

```diff
@@ -0,0 +1,81 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+function LargeScreenTagList({ availableTags, selectedTag, onSelectTag }) {
+  return (
+    <nav aria-label="Filter by tag" className="hidden grid grid-cols-1 gap-2">
+      <a
+        className={`crayons-link crayons-link--block${
+          !selectedTag ? ' crayons-link--current' : ''
+        }`}
+        data-no-instant
+        onClick={onSelectTag}
+        href="/t"
+      >
+        all tags
+      </a>
+      {availableTags.map((tag) => (
+        <a
+          className={`crayons-link crayons-link--block${
+            selectedTag === tag ? ' crayons-link--current' : ''
+          }`}
+          data-no-instant
+          data-tag={tag}
+          onClick={onSelectTag}
+          key={tag}
+          href={`t/${tag}`}
+        >
+          #{tag}
+        </a>
+      ))}
+    </nav>
+  );
+}
+
+/**
+ *
+ * @param {object} props
+ * @param {Array<string>} props.availableTags A list of available tags.
+ * @param {string} [props.selectedTag=''] The currently selected tag.
+ * @param {function} props.onSelectTag A handler for when the selected tag changes.
```

Comment:
nitpick (blocking) - we're missing the `isMobile` prop here

Reply:
done